optionals/compat_sog/CfgAmmo/explosives.hpp

```diff
@@ -61,3 +62,97 @@ class vn_mine_punji_03_ammo: vn_mine_punji_01_ammo {
         };
     };
 };
+
+class vn_mine_punji_04_ammo: APERSBoundingMine_Range_Ammo {
+    EGVAR(minedetector,detectable) = 0;
+
+    hit = QUOTE(call compile getText (configFile >> 'CfgAmmo' >> 'vn_mine_punji_01_ammo' >> 'GVAR(hit)'));
+    GVAR(hit) = QUOTE([ARR_2(0,1)] select isNull (configFile >> 'CfgPatches' >> 'ace_medical'));
+
+    indirectHit = QUOTE(call compile getText (configFile >> 'CfgAmmo' >> 'vn_mine_punji_01_ammo' >> 'GVAR(indirectHit)'));
+    GVAR(indirectHit) = QUOTE([ARR_2(0,10)] select isNull (configFile >> 'CfgPatches' >> 'ace_medical'));
+
+    GVAR(spikesOffset)[] = {0, 0, 1.8};
+    GVAR(spikesCheckSelection) = "head";
+    GVAR(spikesCheckRadius) = 1;
+
+    class EventHandlers {
+        class ADDON {
+            AmmoHit = QUOTE(call FUNC(handlePunjiTrapTrigger));
+        };
+    };
+};
+
+class vn_mine_punji_05_ammo: vn_mine_punji_04_ammo {
+    GVAR(spikesOffset)[] = {0, 0, 0};
+    GVAR(spikesCheckSelection) = "";
+
+    class EventHandlers {
+        class ADDON {
+            AmmoHit = QUOTE(call FUNC(handlePunjiTrapTrigger));
+        };
+    };
+};
+
+class APERSMine_Range_Ammo;
+class vn_mine_bike_ammo: APERSMine_Range_Ammo {
+    EGVAR(explosives,explosive) = "vn_mine_bike_ammo_scripted";
+};
+
+class vn_mine_cartridge_ammo: APERSMine_Range_Ammo {
+    EGVAR(explosives,explosive) = "vn_mine_cartridge_ammo_scripted";
+
+    // bump range and damage slightly, default values do not work well with ACE Medical
+    indirectHit = 2;
+    indirectHitRange = 0.7;
```

Comment:
TODO this should be conditional depending on if ACE Medical is loaded, like punji traps changes.